src/app/shared/dockstore.model.ts

```diff
@@ -49,4 +49,6 @@ export class Dockstore {
   static readonly GITLAB_AUTH_URL = 'https://gitlab.com/oauth/authorize';
   static readonly GITLAB_CLIENT_ID = 'fill_this_in';
   static readonly GITLAB_REDIRECT_URI = Dockstore.LOCAL_URI + '/auth/' + Provider.GITLAB;
+
+  static readonly CWL_VISUALIZER_URI = 'https://view.commonwl.org';
```

Comment:
If it's a constant that rarely changes and only used in one component and service, consider moving it out of this file.  Otherwise, we will have to modify compose_setup's dockstore.model.ts template.

Reply:
I think this is actually fine, I think it fits in along with the gitlab, github, etc.